packages/mui-material/src/ListItem/ListItem.js

```diff
@@ -99,25 +99,29 @@ export const ListItemRoot = styled('div', {
     },
   }),
   [`&.${listItemClasses.focusVisible}`]: {
-    backgroundColor: theme.palette.action.focus,
+    backgroundColor: (theme.vars || theme).palette.action.focus,
   },
   [`&.${listItemClasses.selected}`]: {
-    backgroundColor: alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity),
+    backgroundColor: theme.vars
+      ? `rgba(${theme.vars.palette.primary.mainChannel} / ${theme.vars.palette.action.selectedOpacity})`
+      : alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity),
     [`&.${listItemClasses.focusVisible}`]: {
-      backgroundColor: alpha(
-        theme.palette.primary.main,
-        theme.palette.action.selectedOpacity + theme.palette.action.focusOpacity,
-      ),
+      backgroundColor: theme.vars
+        ? `rgba(${theme.vars.palette.primary.mainChannel} / calc(${theme.vars.palette.action.selectedOpacity} + ${theme.vars.palette.action.focusOpacity}))`
```

Comment:
It's great that we can even use `calc` for calculating the opacity 👍

Reply:
Love it:

<img src="https://user-images.githubusercontent.com/4512430/166459610-56e5a645-25e3-4df3-bc44-7a78f126b3ba.png" width="350px" />